app/lib/server/server.go

```diff
@@ -180,7 +189,8 @@ func InteractiveConfig(binaryName string) {
 		log.Fatal("Impossible to parse public key:", err)
 	}
 
-	server := config.NewServerToml(network.Suite, public, publicAddress)
+	log.Print(publicAddress)
+	server := config.NewServerToml(network.Suite, public, publicAddress, cert)
```

Comment:
just to make it ultra crystal clear: by making this taking a cert we are going in the direction of "everyone" uses TLS. It's good but we then don't worry about backward compatibility. Which for the moment I don't want to since it changes so much. Just to saying the implicit out loud... :D